src/StackExchange.Exceptional.AspNetCore/ExceptionalBuilderExtensions.cs

```diff
@@ -32,11 +34,10 @@ public static IApplicationBuilder UseExceptional(this IApplicationBuilder builde
         /// <param name="builder">The <see cref="IApplicationBuilder"/> instance this method extends.</param>
         /// <param name="applicationName">Application name for this error log.</param>
         /// <exception cref="ArgumentNullException"><paramref name="builder"/> or <paramref name="applicationName"/> is <c>null</c>.</exception>
-        public static IApplicationBuilder UseExceptional(this IApplicationBuilder builder, string applicationName)
+        public static IApplicationBuilder UseExceptional(this IApplicationBuilder builder, IConfiguration configuration, string applicationName)
```

Comment:
This is not the best approach. Agreed. One of the possible solutions that I can think of is instantiating `ConfigSettings` in `ctor` of `ExceptionalMiddleWare`. 

So basically we would add this call in the Startup class if the user likes to use the Configuration for exceptional:

```csharp
services.Configure<ConfigSettings>(Configuration.GetSection("Exceptional")
```

We could then inject `ConfigSections` in `ExceptionMiddleware` ctor and call `Populate(settings)` in there.